Session_3/fill_array.js

```diff
@@ -0,0 +1,14 @@
+// Prefill an Array
+// http://www.codewars.com/kata/prefill-an-array
+
+function prefill(n, v) {
+
+    if (isNaN(parseInt(n)) || n < 0 || n % 1 !== 0)
```

Comment:
avoid multiple if else for returns

if(a === b){
return a;
}
return b:

is equivalent to 
if(a === b){
return a;
} else {
return b:
}